src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -1171,7 +1171,7 @@ private void ReadFromBuffer(Span<byte> buffer)
             Debug.Assert(count <= _readLength - _readOffset);
 
             if (NetEventSource.IsEnabled) Trace($"Copying {count} bytes to stream.");
-            await destination.WriteAsync(_readBuffer, _readOffset, count, cancellationToken).ConfigureAwait(false);
+            await destination.WriteAsync(new ReadOnlyMemory<byte>(_readBuffer, _readOffset, count), cancellationToken).ConfigureAwait(false);
```

Comment:
Nit: could use RemainingBuffer here

Reply:
`RemainingBuffer` is `new ReadOnlyMemory<byte>(_readBuffer, _readOffset, _readLength - _readOffset)` where this is `new ReadOnlyMemory<byte>(_readBuffer, _readOffset, count)`.  I could do `RemainingBuffer.Slice(0, count)`, but I figured why incur the extra slicing when I could just create the right size to begin with.